.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+Pleas give a clear and concise description of what the bug is here.
+
+**To Reproduce**
+If you can describe the steps needed to reproduce here that will help us squash the bug faster.  If you can reproduce the bug using simple code, please put that code here.
```

Comment:
Optional: Since you mention code samples, consider adding that github supports markdown and perhaps linking to https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax or somesuch.

Reply:
done